container-chains/templates/frontier/node/src/service.rs

```diff
@@ -202,6 +269,27 @@ async fn start_node_impl(
         })
         .await?;
 
+    let overrides = crate::rpc::overrides_handle(client.clone());
```

Comment:
Just a comment on storage overrides. If you want to disable it and use runtime api, if you want to mantain a performant server you will need to increase `runtime_cache_size` to a value as high as possible to avoid compiling `VersionedRuntime`s onthefly. This shouldn't be an issuein the beginning but I guess something to take into consideration.